### ISSUE TITLE
Automatic reset of core and video memory overclocking. Авто сброс разгона ядра и видеопамяти.

### ISSUE THREAD

**Issue by Makimfd** [rigelminer/rigel#159]
Hello, could you remove the auto reset for overclocking the video core and video memory, which appeared, it seems, with the release of rigel-1.14.2-win (I could be wrong). I use Windows 10 and have GTX 1660, 1660 Super, 1660 Ti series GPUs. And also RTX series 2060 super, 3060 12GB. 70% of my video cards are from different manufacturers (Palit, MSI, Asus, Ino3D), 3 manufacturers of video memory chips (Samsung, Micron, Hyniks). This is why it is very convenient for me to configure overclocking of video cards using the MSI Autoburner program. With the release of rigel-1.14.2-win (it seems to me, I could be wrong), at the start of any algorithm (also dual and triple mining), the core and memory overclocking is reset to stock values, the power limit and the cooler speed setting are not reset. Perhaps I didn’t understand it well, I don’t write the overclocking settings in the bat file, but the overclocking is reset, now I’m using rigel-1.12.2-win. If there is a simple solution to my problem, by type of command entry or the like. Please help me. Thank you in advance!!!

Дублирую на русском языке. 
Здравствуйте, не могли бы вы убрать авто сброс разгона видеоядра и видеопамяти, которая появилась, кажется с релизом rigel-1.14.2-win (могу ошибаться). Я использую виндовс 10, и у меня графические процессоры серии гтх 1660, 1660 супер, 1660 ти. А так же ртх серии 2060 супер, 3060 12гб. 70% моих видеокарт разных производителей (палит, мси, асус, ино3д), 3 производителя чипов видеопамяти (самсунг, микрон, хайникс). Именно по этому мне очень удобно настраивать разгон видеокарт по средствам программы мси автобёрнер. С выходом релиза rigel-1.14.2-win (как мне кажется, могу ошибаться), при старте любого алгоритма (так же дуал и трипл майнинг) разгон ядра и памяти сбрасывается на стоковые значения, павер лимит и настройка оборотов куллера не сбрасывается. Возможно я плохо разобрался, настройки разгона в бат файл я не прописываю, но разгон сбрасывается, сейчас я использую rigel-1.12.2-win. Если есть простое решение моей проблемы, по типу ввода команды или т.п. пожалуйста помогите мне. Заранее благодарю!!!

**Comment by Makimfd** [comment]
I think you misunderstood me a little. When you start the miner, the memory and core overclocking are reset to standard values, not while loading the dag file, but when you start the bat file and the rig continues to work with the standard core and memory frequencies. This command did not help --dag-reset-mclock off. I will attach a photo, on it you can see the core and memory overclocking reset.
![photo_2024-03-21_20-12-44](https://github.com/rigelminer/rigel/assets/98459065/6548172c-af98-4f6f-aa5a-e20d9e531c23)

**Comment by Makimfd** [comment]
Thank you, you helped me a lot!

**Comment by rigelminer** [comment]
Based on the screenshot above you haven't applied `--dag-reset-mclock off`, the miner still resets it:
 
![image](https://github.com/rigelminer/rigel/assets/119491468/4ff91d18-a358-4175-ace3-0c16bcea80e3)

**Comment by Makimfd** [comment]
That's right, screenshot without using the --dag-reset-mclock off command. Later I launched and added the file --dag-reset-mclock off. Now it works without problems. Thank you for your help. And for your amazing miner!
![photo_2024-03-21_21-46-27](https://github.com/rigelminer/rigel/assets/98459065/d921565f-d42b-4b94-bf59-3039d1d7cb1f)

**Comment by rigelminer** [comment]
Ah, good! Glad I could help :) Thanks.
